app/views/tasks/index.html.erb

```diff
@@ -0,0 +1,24 @@
+<h1><%= link_to("Add a new task here...", tasks_new_path) %></h1>
+
+<h2>Pending Task List:</h2>
+    <% @tasks.each do |t| %>
+        <p>
+            <% if format_complete(t) != "Yes!" %>
+                <%= "#{t[:action]} #{t[:description]}" %>
+                <%= link_to("details", task_path(t[:id])) %>
```

Comment:
Dot notation is also preferring in the views: `t.id` and `t.description` for example